src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * <code>duke.Storage</code> handles all file read and writes.
+ */
+public class Storage {
+    final static String TASKS_TEXT_FILE_PATH = "../../tasks.txt";
+
+    /**
+     * Loads the information from the file "duke.tasks.txt" into a
+     * <code>List</code>. Since each task from the file is
+     * comma seperated, it splits each string into a string array using ","
+     * as a delimiter. Then it creates objects of the child classes of
+     * <code>duke.tasks.Task</code> according to the information provided.
+     * @return a list of duke.tasks as a <code>List</code>
+     * @throws DukeException if there is an IOexception while creating a new empty file
+     */
+    public List<Task> load() throws DukeException {
+        List<Task> tasks = new ArrayList<>();
+        
+        try {
+            File f = new File(TASKS_TEXT_FILE_PATH);
+            Scanner sc = new Scanner(f);
+            
+            while (sc.hasNext()) {
+                String taskInfo = sc.nextLine();
+                String[] arr = taskInfo.split(",");
+                switch (arr[0]) {
+                    case "T":
```

Comment:
Should the case block have 4 less indentation spaces? I noticed this issue in several other places too.

Reply:
Oops, I forgot to change the settings on my IDE. Thanks for highlighting the issue.